### ISSUE TITLE
Europe/Berlin vs Europe/Stuttgart

### ISSUE THREAD

**Issue by aep** [evq/utz#1]
i'm not quite sure why CET is named "Europe/Stuttgart", it should be Europe/Berlin

**Comment by evq** [comment]
@aep I assume you're mean the alias I'm including in `zone.c` which is `Stuttgart` not `Europe/Stuttgart`.

This is a consequence of my use of the default android timezone aliases. I needed to constrain the timezone aliases available for space reasons, if you have suggestions for a better list of aliases I'd love to swap it out!

Anyway, humorously enough I had previously attempted to fix the android aliases in an "impartial" way by incorporating information from wikipedia on the world's most populated major metropolitan areas. But there was actually a bug in my code for this that ended up leaving Berlin and a few other cities out. So this should be fixed in 016d19f.

Let me know if that fix works for you or if I misinterpreted. Thanks!
